examples/web-chat/src/hooks.ts

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from "react";
+import { generate } from "server-name-generator";
+import { Message } from "./Message";
+import { Decoder } from "@waku/core/lib/message/version_0";
```

Comment:
Should we re-export that in `@waku/react`?

Reply:
I think no because `@waku/react` is a helper for easier usage of `js-waku` for now.